chipsec/chipset.py

```diff
@@ -656,6 +661,23 @@ def init_cfg_xml(self, fxml, code, pch_code):
                     self.Cfg.CONTROLS[ _name ] = _control.attrib
                     if logger().DEBUG: logger().log( "    + %-16s: %s" % (_name, _control.attrib) )
 
+    def init_cfg_bus( self ):
+        if logger().DEBUG: logger().log( '[*] loading device buses..' )
+        _devices = self.pci.enumerate_devices()
+        for _device in self.Cfg.CONFIG_PCI:
+            device = self.Cfg.CONFIG_PCI[_device]
+            _vid  = device.get( 'vid', None )
+            _did  = device.get( 'did', None )
+            if (_vid and _did):
+                _bus = []
+                did  = [int(_,16) for _ in _did.split(',')]
+                for _dev in _devices:
```

Comment:
the naming between _devices, _device, device, _dev is confusing, is there a better naming convention which would help?

Reply:
Due to overloading of the XML attributes and BDF.  I can add clarifying labels.

Reply:
thanks!